tethys_gizmos/gizmo_options/base.py

```diff
@@ -38,7 +39,52 @@ def __init__(self, attributes={}, classes=''):
         self.attributes = attributes
         self.classes = classes
 
+    @staticmethod
+    def get_tethys_gizmo_js():
```

Comment:
Also, I was thinking to remove those dependencies and move them into local gizmo js/css files.  My thinking is to make wrappers for AJAX stuff.